loops-and-arrays.js

```diff
@@ -53,3 +54,12 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+const exponent = 3
+const squares = [30, 2, 8, 24, 11];
+const uitkomst = [0, 0, 0, 0, 0];
```

Comment:
Hi Thijs! :) 
Ik zou hier in plaats van uitkomst "result" gebruiken ivm code in Engels schrijven - inhoud in Nederlands